site/link-checker-config.json

```diff
@@ -19,5 +19,8 @@
       "pattern": "^../../",
       "replacement": "{{BASEURL}}/site/docs/"
     }
-  ]
-}
\ No newline at end of file
+  ],
+  "retryOn429": true,
+  "retryCount": 5,
+  "fallbackRetryDelay": "120s"
```

Comment:
can we reduce the retries? I think with that retry the last CI run took 48 mins, which is too long IMO

Reply:
I don't think it will work anyway. I'm not exploring this approach now.